src/conversions.jl

```diff
@@ -392,7 +392,7 @@ function convert{T}(::Type{Vector{T}}, o::PyObject)
     py2array(T, Array{pyany_toany(T)}(len), o, 1, 1)
 end
 
-convert(::Type{Array}, o::PyObject) = py2array(PyAny, o)
+convert(::Type{Array}, o::PyObject) = map(identity, py2array(PyAny, o))
```

Comment:
It does, but I think it's an acceptable tradeoff. I had a different version that tried really hard to avoid that, but it ended up just being more complicated and hard to understand. At some future point, the compiler might even be able to optimize this away.